edbterraform/data/terraform/gcloud/modules/gke/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "region" {
+  value = var.machine.spec.region
+}
+output "kubernetes_cluster_name" {
+  value       = google_container_cluster.primary.name
+  description = "GKE Cluster Name"
+}
+output "kubernetes_cluster_host" {
+  value       = google_container_cluster.primary.endpoint
+  description = "GKE Cluster Host"
+}
```

Comment:
These outputs will be used for `server.yml` file and `terraform output`
If we could add variables for `public_ip` and `private_ip` and any others that should be known.

Example: https://github.com/EnterpriseDB/edb-terraform/blob/main/edbterraform/data/terraform/gcloud/modules/machine/outputs.tf

Reply:
Not needed since ip's are not utilized in same manner as vm's

Reply:
to keep things consistent with the other module outputs, would it be appropriate to define `kubernetes_cluster_host` as a `public_ip` or `private_ip`? Depending on the endpoint being public or not, one would be defined and one set to null?

Reply:
Not needed to expose the k8s cluster host because it is not used for managing the cluster.

See example: 
https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#default_cluster_kubectl